src/pages/map/MapPage.tsx

```
@@ -104,8 +104,64 @@ const MapContent = () => {
+      
+      // 스크롤 가능한 영역이나 가로 스크롤 영역은 허용
+      const isScrollableArea = target.closest('[data-scrollable="true"]') || 
+                              target.closest('.overflow-x-auto') ||
+                              target.closest('.scrollbar-hide') ||
+                              target.closest('[class*="overflow-x"]');
+      
+      if (!isScrollableArea) {
+        // 터치 이동이 주로 세로 방향인지 확인
+        const touch = e.touches[0];
+        if (touch) {
+          const deltaX = Math.abs(touch.clientX - (touch.target as any).startX || 0);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)